tests/serverSideProps.test.ts

```diff
@@ -0,0 +1,60 @@
+import { IncomingMessage, ServerResponse } from "http";
+import fetch from "jest-fetch-mock";
+import { enableFetchMocks } from "jest-fetch-mock";
+import { Socket } from "net";
+
+import { getServerSideProps } from "../src/pages/index";
+
+enableFetchMocks();
+
+describe("getServerSideProps", () => {
+  beforeEach(() => {
+    fetch.resetMocks();
+  });
+
+  const mockedContext = {
+    req: new IncomingMessage(new Socket()),
+    res: new ServerResponse(new IncomingMessage(new Socket())),
+    query: {},
+  };
```

Comment:
```suggestion
  const mockedRequest = new IncomingMessage(new Socket())
  const mockedContext = {
    req: mockedRequest,
    res: new ServerResponse(mockedRequest),
    query: {},
  };
```

Reply:
That's weird.. that's the first thing I did when I mocked the request, but I had ts freaked out about it. I probably made a mistake.